internal/lib/lib.go

```diff
@@ -488,6 +488,7 @@ func InformersToRegister(oclient *oshiftclient.Clientset, kclient *kubernetes.Cl
 		utils.EndpointInformer,
 		utils.SecretInformer,
 		utils.ConfigMapInformer,
+		utils.PodInformer,
```

Comment:
Should this be added only if serviceType is NodePortLocal?

Reply:
Yeah, added a check